schemas/data/page-views.schema.json

```diff
@@ -5,7 +5,7 @@
     "you may not use this file except in compliance with the License. You may obtain a copy",
     "of the License at https://creativecommons.org/licenses/by/4.0/"
   ],
-  "$id": "https://ns.adobe.com/xdm/data/metrics/web/page-views",
+  "$id": "https://ns.adobe.com/xdm/metrics/web/page-views",
```

Comment:
Now it's a breaking change. I suggest to leave it in `data/metrics`.

Reply:
Because this is moving in the hierarchy it is already a breaking change. I strongly recommend we move tot he URIs we want at this point. The teams are already absorbing a number of breaking changes so this can be absorbed at this point.

Reply:
+1 on this, moving the fields to a different parent is already breaking, better we do it now versus having another breaking change in the same area later.  Can we agree on putting everything in "https://ns.adobe.com/xdm/metrics"?

Reply:
If we move it to `xdm/metrics`, we will have to move all files from the `schemas/data` folder to the `schema/metrics` folder and introduce this new folder. I just don't think that's worth it.